CHANGES.md

```diff
@@ -1,5 +1,28 @@
 # Changes
 
+## stable-2.13.2
+
+This stable release fixes an incompatibility issue with the AWS CNI addon in EKS
+that was forbidding pods to acquire networking after scaling up nodes (thanks
+@frimik!). It also includes security updates for dependencies.
+
+* CNI
+  * Fixed incompatibility issue with AWS CNI addon in EKS, that was forbidding
+    pods to acquire networking after scaling up nodes. (thanks @frimik!)
+
+* CLI
+  * Fixed `linkerd uninstall` issue for HttpRoute
```

Comment:
Might be good to elaborate here.  Something like, added a missing label to the HttpRoute CRD so that to ensure it can be removed by the `linkerd uninstall` command?